### PR TITLE
Update Message Subscriber sample widget

### PR DESCRIPTION
**Pull request by shawnmgoulet**
A query action fix within the Message Subscriber sample.

### CODE REVIEW

**Comment by shawnmgoulet** [comment]
@gavinr @qlqllu - I feel like this should go into the `master` branch as the issue is presented at `1.9`. This PR includes:
1. Adding the `filterMessageDescription` method to `query-action.ts`
2. linting `query-action.ts`
3. linting other files within the widget directory

File: widgets/message-subscriber/src/actions/query-action.ts [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
-    return 'actions/query-action-setting';
+  getSettingComponentUri (messageType: MessageType, messageWidgetId?: string): string {
+    return 'actions/query-action-setting'
   }
 
-  onExecute(message: Message, actionConfig?: any): Promise<boolean> | boolean{
+  onExecute (message: Message, actionConfig?: any): Promise<boolean> | boolean {
     let q = `${actionConfig.fieldName} = '${message}'`
-    switch(message.type){
+    switch (message.type) {
```
Comment (qlqllu) [inline widgets/message-subscriber/src/actions/query-action.ts:23]: As you remove the `StringSelectionChange` from the `filterMessageDescription`, I think you can remove this switch as here as well. The `StringSelectionChange` is not documented and is not used anywhere now.

**Comment by qlqllu** [comment]
@shawnmgoulet Thanks for working on this fix! It'll be easy to review if you split the bug fix from the lint fix.

File: widgets/message-subscriber/src/actions/query-action.ts (hunk 1 above)
Comment (shawnmgoulet) [reply widgets/message-subscriber/src/actions/query-action.ts:23]: Thanks, will remove `StringSelectionChange` in upcoming commit.

**Comment by shawnmgoulet** [comment]
> @shawnmgoulet Thanks for working on this fix! It'll be easy to review if you split the bug fix from the lint fix.

I am splitting linting and code refactors into separate commits.

File: widgets/message-subscriber/src/actions/query-action.ts (hunk 1 above)
Comment (shawnmgoulet) [reply widgets/message-subscriber/src/actions/query-action.ts]: Commit https://github.com/Esri/arcgis-experience-builder-sdk-resources/pull/121/commits/1b2ba8c0eacfe21d7ced22aa22451a69a508505a resolves this.

**Pull request closed without merging**